x/cert/types/cert.pb.go

```diff
@@ -690,77 +690,77 @@ func init() {
 func init() { proto.RegisterFile("shentu/cert/v1alpha1/cert.proto", fileDescriptor_14e43b05a8c34048) }
 
 var fileDescriptor_14e43b05a8c34048 = []byte{
```

Comment:
Could it be that the pb file has changed due to using a different protoc version? 
Should we explain the version used later, so that the generated pb files are consistent.